jupyterhub_traefik_proxy/etcd.py

```diff
@@ -75,8 +79,22 @@ def _default_client(self):
                     self.etcd_traefik_prefix + "entrypoints/http/address",
                     ":" + str(urlparse(self.public_url).port),
                 ),
-                KV.put.txn(self.etcd_traefik_prefix + "api/dashboard", "false"),
-                KV.put.txn(self.etcd_traefik_prefix + "api/entrypoint", "http"),
+                KV.put.txn(
+                    self.etcd_traefik_prefix + "entrypoints/auth_api/address",
+                    ":" + self.traefik_auth_api_port,
+                ),
+                KV.put.txn(
+                    self.etcd_traefik_prefix
+                    + "/entrypoints/auth_api/auth/basic/users/0",
+                    "test:$apr1$H6uskkkW$IgXLP6ewTrSuBkTrqE8wj/",
```

Comment:
This should come from config, e.g. `self.traefik_api_basic_auth_username` and `self.traefik_api_basic_auth_password`. The password should then be base64-encoded before being added here.